app/core-components/navigation/NavigationBar.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+import * as React from 'react';
+import { translate } from 'react-i18next';
+import type { TranslatorProps } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { Image, Menu } from 'semantic-ui-react';
+
+type Props = TranslatorProps & { };
+
+// TODO: pass t function to MenuRight
+const MenuRight = (): React.Node => {
+  return (
+    <Menu.Menu position="right">
+      <Menu.Item>
+        <Link to="/Library">
+          <Image size="mini" src="/assets/images/icons/book.png" />
```

Comment:
Images (assets) shouldn't be referenced directly. If the app is hosted on a subdirectory, the references will be wrong. Instead, `import` and reference them using webpack.

Reply:
Even better, use font icons for this: https://react.semantic-ui.com/elements/icon . That way they won't look pixelated no matter what size you make them.

Reply:
It was a temp fix because the icons weren't working